app/src/main/java/com/duckduckgo/app/generalsettings/GeneralSettingsActivity.kt

```diff
@@ -75,6 +85,18 @@ class GeneralSettingsActivity : DuckDuckGoActivity() {
         setContentView(binding.root)
         setupToolbar(binding.includeToolbar.toolbar)
 
+        binding.maliciousLearnMore.addClickableSpan(
+            textSequence = getText(R.string.maliciousSiteSettingLearnMore),
+            spans = listOf(
+                "learn_more_link" to object : DuckDuckGoClickableSpan() {
+                    override fun onClick(widget: View) {
+                        viewModel.maliciousSiteLearnMoreClicked()
+                    }
+                },
+            ),
+        )
+        binding.maliciousDisabledMessage.setTextColor(getColorFromAttr(com.duckduckgo.mobile.android.R.attr.daxColorDestructive))
```

Comment:
can we do this from xml?

Reply:
I tried to, and it wouldn't take effect for some weird reason. I'll poke it again and see if I can get it to work.

Reply:
Aha! Secondary text distinction got in the way.